### PR TITLE
@alloy->fetch ALL works in show on artwork view

### PR DESCRIPTION
**Pull request by 1aurabrown**
fixes https://github.com/artsy/eigen/issues/472

screen shot 2015-06-29 at 2 01 30 pm

### CODE REVIEW

**Comment by alloy** [comment]
Aha, I must have misunderstood the use of pagination in the ticket for on-demand pagination (e.g. scrolled to bottom).

I guess there are no shows with massive amounts of works, huh?

**Comment by 1aurabrown** [comment]
The api is paginated. Probably most shows are 1-2 pages

On Mon, Jun 29, 2015 at 14:21, artsy/eigen
<reply@reply.github.com>
wrote:
Aha, I must have misunderstood the use of pagination in the ticket for on-demand
pagination (e.g. scrolled to bottom).

I guess there are no shows with massive amounts of works, huh?

—
Reply to this email directly or view it on GitHub
[https://github.com/artsy/eigen/pull/560#issuecomment-116783464] .[https://github.com/notifications/beacon/ADBlTnsdpc0kMr-s2RHjq6jcHYeGE8Xgks5oYYRGgaJpZM4FOU5C.gif]

**Comment by alloy** [comment]
Can you add a CHANGELOG?

And are you planning on adding tests?

**Comment by alloy** [comment]
> Probably most shows are 1-2 pages

Gotcha, good call then :+1:

File: Gemfile.lock [hunk 1]
```diff
@@ -247,6 +247,3 @@ DEPENDENCIES
   second_curtain
   shenzhen
   xcpretty
-
```
Comment (orta) [inline Gemfile.lock]: you should upgrade bundler., it would have complained when you did this: `gem install bundler`

**Comment by alloy** [comment]
Superbelicious :+1:

**Pull request merged**